homeassistant/components/sensor/integration.py

```diff
@@ -49,7 +55,9 @@
     vol.Optional(CONF_ROUND_DIGITS, default=DEFAULT_ROUND): vol.Coerce(int),
     vol.Optional(CONF_UNIT_PREFIX, default=None): vol.In(UNIT_PREFIXES),
     vol.Optional(CONF_UNIT_TIME, default='h'): vol.In(UNIT_TIME),
-    vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string
+    vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_INTEGRATION_METHOD, default=TRAPEZOIDAL_METHOD):
+        vol.In(INTEGRATION_METHOD)
```

Comment:
Tip: Put an `,` at the end here. Then the diff view will be nicer and if you need to create a patch release it will be easier with merge conflicts

Reply:
👍